### ISSUE TITLE
Positional encoding are already part of the input

### ISSUE THREAD

**Issue by Atlis** [lucidrains/perceiver-pytorch#25]
Hello!
First of all, thank you for this implementation.

My inputs already have the proper positional encoding as part of the channel axis.
Would it be possible to add a feature to deactivate the default implementation of the positional encoding?
 
Thank you!

**Comment by Atlis** [comment]
In the mean time I commented those lines:
```python
        # axis_pos = list(map(lambda size: torch.linspace(-1.,
        #                                                 1., steps=size, device=device), axis))
        # pos = torch.stack(torch.meshgrid(*axis_pos), dim=-1)
        # enc_pos = fourier_encode(
        #     pos, self.max_freq, self.num_freq_bands, base=self.freq_base)
        # enc_pos = rearrange(enc_pos, '... n d -> ... (n d)')
        # enc_pos = repeat(enc_pos, '... -> b ...', b=b)

        # concat to channels of data and flatten axis

        #data = torch.cat((data, enc_pos), dim=-1)
```
And rewrote:
```python
#input_dim = input_axis * ((num_freq_bands * 2) + 1) + input_channels
input_dim = input_channels
```

Does that make sense?

Thank you.

**Comment by lucidrains** [comment]
@Atlis yup, let me add that now..

**Comment by lucidrains** [comment]
@Atlis ok, done in 0.2.0

**Comment by Atlis** [comment]
Thanks!
